lib/oxidized/model/alvarion

```diff
@@ -0,0 +1,13 @@
+class Alvarion < Oxidized::Model
+
+  # Used in Alvarion wisp equipment
+
+  # Run this command as an instance of Model so we can access node
+  pre do
```

Comment:
maybe we should have more descriptive alias for pre or post, which would have same behaviour for now, just use-cases where argument themselves need to see instance context.